src/main/resources/application.yml

```diff
@@ -44,6 +44,7 @@ queueconfig:
   case-update-topic: event_case-update
   uac-update-topic: event_uac-update
   deactivate-uac-topic: event_deactivate-uac
+  sms-fulfilment-subscription: rm-internal-sms-fulfilment_case-processor
```

Comment:
🎨 I would probably keep the subscriptions grouped together, personally... although I see that it hasn't been followed consistently (probably by me) anyway